tests/test_logger.py

```diff
@@ -307,3 +307,18 @@ class CustomLogger(AwareLogger):
         assert len(lines) > 1
         assert lines[1].startswith(b'Traceback'), \
             'Expected traceback info in log message'
+
+
+def test_logger_is_enabled_for():
+    """
+    Verify that an AwareLogger can be used like a native logger instance for
+    determining if logging is enabled for a specific log level.
+    """
+    logger = logging.getLogger()
+    previous_level = logger.level
+    logger.setLevel(999)
```

Comment:
This extra junk in this test is the same reason that the first doctest is failing. If that is resolved, this test can get much simpler, probably.